docs/language/managing_values.rst

```diff
@@ -88,6 +88,10 @@ Here is an example:
         }
     }
 
+On Solana, only the owner can change the balance of its accounts. As a corollary, the functionality of ``send()`` and
+``transfer`` is limited. They only work when the contract's program account owns both the sender and the
+recipient accounts.
```

Comment:
I think it is clearer to first state the limitation, and then the cause. Also, there are many superfluous words. The paragraph should be a note, and possibly merged with the last note in this file. How about:

On Solana, ``send()`` and ``transfer()`` can only transfer native tokens between accounts that are both own by
the current contract's program account, since only the owner can modify account balances. Use the 
:ref:`system instruction library <system_instruction_library>`_ `transfer()` function to avoid this problem.

Reply:
Fixed